olp-cpp-sdk-core/src/cache/DefaultCacheImpl.cpp

```diff
@@ -568,22 +633,46 @@ uint64_t DefaultCacheImpl::MaybeEvictData(leveldb::WriteBatch& batch) {
     it = mutable_cache_lru_->Erase(it);
   }
 
-  // Remove the other elements if needed
+  OLP_SDK_LOG_DEBUG_F(kLogTag,
+                      "EvictExpiredDataPortion(): Evicted successfully, "
+                      "count=%d, evicted=%" PRIu64,
+                      count, evicted);
+  return {count, evicted, false};
+}
+
+DefaultCacheImpl::EvictionResult DefaultCacheImpl::EvictDataPortion(
+    leveldb::WriteBatch& batch) {
+  uint64_t evicted = 0u;
+  auto count = 0u;
+  const auto target_size = kMinDiskUsedThreshold * settings_.max_disk_storage;
```

Comment:
Maybe we should use here MaxDiskUsedThreshold? Because I don't think that we want to remove any data which is not required for removal.

Reply:
It was an original logic: If the size is bigger than k**Max**DiskUsedThreshold * settings_.max_disk_storage _(condition)_, we evict until the size reaches k**Min**DiskUsedThreshold * settings_.max_disk_storage _(target)_. So I didn't change this logic in my commit